### ISSUE TITLE
Do we need to train on training set and validation set both?

### ISSUE THREAD

**Issue by zhaobinNF** [karstenBehrendt/unsupervised_llamas#13]
Thanks for the high-quality dataset. Do we need to train on both training set and validation set and then evaluating our results on testing set? Or just training on training set and evlauate on testing set.

**Comment by karstenBehrendt** [comment]
Hi @zhaobinNF - You can choose to train on any part of the dataset that you want to. Generally, the validation test allows you to check if you have been overtraining to some extent and the alternative is to (partially) include the validation set in your training as well for a larger training corpus. So, either is fine from my perspective. Feel free to use anything that you want.
